lldb/source/Plugins/TypeSystem/Swift/StoringDiagnosticConsumer.h

```diff
@@ -198,6 +199,13 @@ class StoringDiagnosticConsumer : public swift::DiagnosticConsumer {
       std::string &s = os.str();
       formatted_text = !s.empty() ? std::move(s) : std::string(text);
     }
+    if (info.Kind == swift::DiagnosticKind::Remark &&
+        info.ID == swift::diag::module_loaded.ID) {
+      // Divert module import remarks into the logs.
+      LLDB_LOG(GetLog(LLDBLog::Types), "{0} Module import remark: {2}",
+               m_ast_context.GetDescription(), __FUNCTION__, formatted_text);
```

Comment:
how come you include `__FUNCTION__` as an argument?

Reply:
Thanks! That's from an earlier draft.